torch/csrc/jit/ir.cpp

```diff
@@ -786,6 +786,7 @@ Node* Node::insertAfter(Node * n) {
   JIT_ASSERT(!inBlockList() && n->inBlockList());
   JIT_ASSERT(n->owningBlock());
   this->owning_block_ = n->owningBlock();
+  this->owning_block_->topological_index_.insertAfter(n, this);
```

Comment:
Only insertAfter is ever used, so I am not sure why there is an implementation of insertBefore as well.